spezi-module/contact/src/androidTest/java/edu/stanford/spezikt/spezi_module/contact/ContactScreenTest.kt

```diff
@@ -0,0 +1,109 @@
+package edu.stanford.spezikt.spezi_module.contact
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Call
+import androidx.compose.material.icons.filled.Email
+import androidx.compose.material.icons.filled.Info
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import edu.stanford.spezikt.spezi_module.contact.model.Contact
+import edu.stanford.spezikt.spezi_module.contact.model.ContactOption
+import edu.stanford.spezikt.spezi_module.contact.model.ContactOptionType
+import edu.stanford.spezikt.spezi_module.contact.repository.ContactRepository
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.whenever
+import java.util.UUID
+
+class ContactScreenTest {
+
+    private val mockContactRepository: ContactRepository = mock()
+
+    @get:Rule
+    val composeTestRule = createComposeRule()
+
+
+    @Before
+    fun setup() {
+        whenever(mockContactRepository.getContact()).thenReturn(createContact())
+    }
+
```

Comment:
I generally use the following setup to test composables. For every composable, I would define the following:

```kotlin
fun OnContactScreen(composeTestRule: ComposeContentTestRule, testScope: ContactScreen.() -> Unit) {
    ContactScreen(composeTestRule).apply(testScope)
}

class ContactScreen(private val composeTestRule: ComposeContentTestRule) {
    private val nameTag = "We have to set the tag at the name Text at ContactScreen via the modifier"

    fun assertNameExists() {
        composeTestRule.onNodeWithTag(nameTag).assertExists()
    }

    fun assertNameIs(name: String) {
        composeTestRule.onNodeWithTag(nameTag).assertTextEquals(name)
    }
}
```

and then the tests would look like this:

```kotlin
@Test
fun example_test() {
    composeTestRule.setContent {
        ContactScreen(createContact())
    }

    OnContactScreen(composeTestRule) {
        assertNameExists()
        assertNameIs("expected name")
    }
}
```

It offers some consistency and a DSL like API. Let me know what do you think